CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing to Refitter
+
+Thank you for your interest in contributing to Refitter! This document provides guidelines and instructions for contributing to the project.
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [How to Contribute](#how-to-contribute)
+  - [Reporting Issues](#reporting-issues)
+  - [Feature Requests](#feature-requests)
+  - [Pull Requests](#pull-requests)
+- [Development Guidelines](#development-guidelines)
+  - [Code Quality](#code-quality)
+  - [Testing](#testing)
+  - [Documentation](#documentation)
+- [Style Guide](#style-guide)
+
+## Code of Conduct
+
+Please review and adhere to our code of conduct to ensure a positive and inclusive environment for all contributors.
+
+## How to Contribute
+
+### Reporting Issues
+
+If you encounter a bug or issue, please create a GitHub issue with the following information:
```

Comment:
Add details regarding support keys as described in the .github\ISSUE_TEMPLATE\bug_report.md template